lib/js/components/Drawer/DrawerSection/DrawerSection.stories.ts

```diff
@@ -12,25 +18,97 @@ const StoryTemplate: StoryFn<typeof DrawerSection> = (args) => ({
 	setup() {
 		return { ...args };
 	},
+	data() {
+		return {
+			ICONS: Object.freeze(ICONS),
+		};
+	},
+	methods: {
+		infoClicked() {
+			console.log('info clicked');
+		},
+	},
 	template: `
 		<drawer-section
-			:has-divider="hasDivider"
-			:is-expandable="isExpandable"
+			:size="size"
 			:title="title"
+			:supporting-text="supportingText"
+			:eyebrow="eyebrow"
+			:icon-left="ICONS[iconLeft]"
+			:icon-left-color="iconLeftColor"
+			:icon-right="ICONS[iconRight]"
+			:icon-right-color="iconRightColor"
+			:info="info"
+			:divider="divider"
```

Comment:
Let's rename it in SectionHeader to`hasDivider`

Reply:
Some time ago we started to add those `has` and `is` prefixes. It would be nice to make it consistent and aligned with current way of naming props. It is a suggestion, tho